File: src/main/webapp/app/Prosjekt.js
function Prosjekt(data){
    "use strict";
    var self = this;
    self.id = ko.observable();
    self.konsulenter = ko.observableArray();
    self.navn = ko.observable();

    if(data != undefined && data != null){
        self.id(data.id);

        $.map(data.konsulenter, function(konsulentJson){
            self.konsulenter.push(new Konsulent(konsulentJson));
        });

        self.navn(data.navn);
    }
}

File: src/main/java/com/soprasteria/boot/domain/Prosjekt.java
package com.soprasteria.boot.domain;

import org.springframework.stereotype.Component;

import java.util.List;


@Component
public class Prosjekt {
	private int id; 
	private List<Konsulent> konsulenter;
	private String navn;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public List<Konsulent> getKonsulenter() {
		return konsulenter;
	}

	public void setKonsulenter(List<Konsulent> konsulenter) {
		this.konsulenter = konsulenter;
	}

	public String getNavn() {
		return navn;
	}

	public void setNavn(String navn) {
		this.navn = navn;
	}
}
